packages/block-editor/src/components/rich-text/index.js

```diff
@@ -404,9 +403,11 @@ export function RichTextWrapper(
 				// Overridable props.
 				role="textbox"
 				aria-multiline={ ! disableLineBreaks }
-				aria-label={ placeholder }
 				aria-readonly={ shouldDisableEditing }
 				{ ...props }
+				aria-label={
+					bindingsPlaceholder || props[ 'aria-label' ] || placeholder
```

Comment:
is `placeholder` already defaulted on line 337?

Reply:
> is placeholder already defaulted on line 337?

+1 — I think having just `aria-label={ placeholder || props[ 'aria-label' ] }` should be good.

Reply:
The problem is the priority. `props[ 'aria-label' ]` is supposed to override the default placeholder value if it exist for the `aria-label` attribute. And I believe we want to override the `aria-label` prop when a binding exists.

Reply:
@zaguiini, regarding your comment from https://github.com/WordPress/gutenberg/pull/65089#pullrequestreview-2283038208.

> Another issue, though related, is that the Block Bindings API does not respect an existing placeholder attribute. From the back-end, I'm setting a bespoke placeholder:

I think this is what changed the behavior, as placeholder became here a fallback. Now, the question is whether there is always `placeholder` coming from the server. If not, then we should be fine reverting to the old behavior and including the fallback to `bindingsPlaceholder` when the RichText is connected to a source.